src/runtime_src/core/pcie/linux/scan.cpp

```diff
@@ -577,6 +577,34 @@ int pcidev::pci_device::get_partinfo(std::vector<std::string>& info, void *blob)
     return 0;
 }
 
+std::string pcidev::pci_device::sysfs_get_userpf()
```

Comment:
you should just use get_dev() and pass false to it to get user pf.

Reply:
Updated the code